static/widgets/fontscale.ts

```diff
@@ -66,11 +64,26 @@ function makeFontSizeDropdown(elem: JQuery, obj: FontScale, buttonDropdown: JQue
         }
         elem.children().eq(selectedId).trigger('click');
     });
+
+    // ctrl+click handler
+    buttonDropdown.on('click', e => {
+        if (e.ctrlKey) {
+            // This is a hack. It prevents bootstrap's click listener from opening the dropdown
```

Comment:
Not a hack if it's the only way to avoid the default behaviour right?

Reply:
Haha fair enough :) I wish there was a better way to do this, I don't think it has too much potential to cause problems down the road at least